Resources/Prototypes/Entities/Objects/Misc/paper.yml

```diff
@@ -25,9 +25,6 @@
           enum.PaperStatus.Written: paper_words
   - type: Item
     size: 1
-  - type: Tag
-    tags:
-    - Trash
 
 - type: entity
```

Comment:
Not sure if written paper should be trashable.

Reply:
Put paper back to baseitem since it already had the item size. Needs paper bin in future.